.github/workflows/test.yml

```diff
@@ -158,7 +158,7 @@ jobs:
         path: service/build/reports/tests
 
     - name: "Notify QA Slack"
-      if: always() && (steps.set-env-step.outputs.test-env == 'alpha' || steps.set-env-step.outputs.test-env == 'staging')
+      if: failure() && (steps.set-env-step.outputs.test-env == 'alpha' || steps.set-env-step.outputs.test-env == 'staging')
```

Comment:
This one goes to the QA slack. We should probably not change this one, unless QA agrees.
@ungwudik is the QA person I usually work with but he is on vacation until Aug 6.

Reply:
Reverted